### DISCUSSION TITLE
ch02_05. 테라폼 HCL module 실습에서 오류 발생

### DISCUSSION THREAD

**Discussion by Virusuki** [tedilabs/fastcampus-devops#53] [Q&A] [answered]
안녕하세요. 
좋은 강의에 감사드립니다.
CH02_05. 테라폼 HCL 모듈 실습 중에 
https://github.com/tedilabs/fastcampus-devops/tree/main/2-terraform/04-module 여기서 main.tf를 다운받고
terraform init을 하는데 에러가 발생해서 문의드려요...
아래와 같이 sorurce 관련 에러인데 soruce를 대체하는 파일이 있을까요? ㅠㅠ
  source  = "tedilabs/network/aws//modules/route-table"

Error: Failed to download module
│ 
│ Could not download module "route_table__private" (main.tf:83) source code
│ from "git::https://github.com/tedilabs/terraform-aws-network?ref=v0.24.0":
│ error downloading
│ 'https://github.com/tedilabs/terraform-aws-network?ref=v0.24.0': git must be
│ available and on the PATH.

![테라폼_init에러](https://user-images.githubusercontent.com/70243256/155823519-484f5090-0a11-42cb-9386-3a3edeb9a9a1.PNG)

**Comment by posquit0** [comment, marked as answer]
앗.. 불편을 드려 죄송합니다.
테라폼에서 모듈을 가져오는 방법으로 현재 설정된 source는 git을 사용중에 있습니다.
컴퓨터에 우선 git을 설치하시고 다시 시도 부탁드립니다..!! ㅎㅎ
